### DISCUSSION TITLE
Difference between this project and wxc_sdk

### DISCUSSION THREAD

**Discussion by jmrbcu** [kctrey/wxcadm#11] [Q&A] [answered]
Hello guys, I am about to create a service app, and I am trying to figure out what the difference is between this project and **wxc_sdk**: [https://github.com/jeokrohn/wxc_sdk/](https://github.com/jeokrohn/wxc_sdk/)

**Comment by kctrey** [comment, marked as answer]
I am not sure how I missed this, but the key difference is that wcd_sdk is meant to be a 1:1 SDK that is identical to the Webex API. wxcadm actually uses wxc_sdk for some work and probably will use it for everything in the future. This project is a higher-level approach, meant to represent the Webex APIs in ways that admins might think of them rather than the, sometimes confusing, way the APIs actually work. In this project, I try to "abstract" some of the API calls into a single method or property where wxc_sdk will always be 1:1 on the number of methods:endpoints.
